packages/react-server/src/ReactFizzHooks.js

```diff
@@ -545,6 +551,11 @@ function useTransition(): [
   return [false, unsupportedStartTransition];
 }
 
+function useHostTransitionStatus(): FormStatus {
+  resolveCurrentlyRenderingComponent();
+  return NotPending;
+}
```

Comment:
You probably need to move this to FizzConfigDOM since this file isn’t DOM specific which is why you get this error. Needs to be behind a “Config/Binding”.